src/components/Cart/Cart.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import StripeCheckout from 'react-stripe-checkout'
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+
+toast.configure();
+
+const Cart = (props) => {
+  const { cartItems, onAdd, onRemove, onClean, onCreateOrder } = props;
```

Comment:
here you can destructure your props at the top, something like this:

`const Cart = ({
  cartItems,
  onAdd,
  onRemove,
  ...
}) => {
....
`